src/main/webapp/i18n/en/competency.json

```diff
@@ -147,6 +147,32 @@
                 "selfRelation": "You can not create a relation between a competency and itself.",
                 "relationAlreadyExists": "This relation already exists."
             }
+        },
+        "learningPath": {
+            "learningPathButton": "Learning Paths",
+            "manageLearningPaths": {
+                "title": "Learning Path Management",
+                "isDisabled": "Learning Paths are currently disabled for this course.",
+                "enable": "Enable Learning Paths",
+                "enableHint": "The creation of Learning Paths for every student may take a few minutes.",
```

Comment:
does it really take minutes? If so, is this done async?

Reply:
It really doesn't. In the beginning, I thought so, but after some refactoring, this is not the case anymore.